mix.exs

```diff
@@ -32,7 +32,7 @@ defmodule Kitto.Mixfile do
     [{:cowboy, "~> 1.0.0"},
      {:plug, "~> 1.3"},
      {:poison, "~> 3.0"},
-     {:fs, "~> 2.11.0"},
+     {:fs, github: "synrc/fs"},
```

Comment:
What do you think about maybe dropping in https://github.com/falood/exfswatch as a replacement for fs? Seems to have more documentation and tagging.